data_generation/main.py

```diff
@@ -0,0 +1,31 @@
+"""This is entrypoint function that contains creates a prompt for ChatGPT
+and obtains a response (an exam question) from it for a given topic
+"""
+
+
+import os
+import openai
+from datetime import datetime
+
+openai.api_key = os.getenv("OPENAI_API_KEY", None)
+from utils.chatgpt_prompt_utils import get_exam_questions_as_list
+
+def get_Session_id():
+    return datetime.now().strftime("%Y-%m-%d_%H%M%S")
+
+
+def _set_apis(self):
```

Comment:
What is the purpose of this function, its not being used and the code is commented out?